Jenkinsfile

```diff
@@ -14,10 +14,12 @@ elifePipeline {
             // Note: we are using staging source dataset in ci
             //   because some source tables or views may not be present in ci
             try {
-                sh "make IMAGE_TAG=${commit} REVISION=${commit} \
-                    DATA_SCIENCE_SOURCE_DATASET=staging \
-                    DATA_SCIENCE_OUTPUT_DATASET=ci \
-                    ci-build-and-test"
+                timeout(time: 30, unit: 'MINUTES') {
```

Comment:
It is different from the example (which was a parameter to elifePipeline). But this seems to be more standard and readable.

I wonder why the [options](https://www.jenkins.io/doc/book/pipeline/syntax/#options) approach didn't work.

Reply:
It did not work because as I see there is two approach for coding in jenkins file with groovy. One is `declarative pipeline `which has the options section. The other one is `scripted pipeline` which has doesn't now about options. And we are using scripted pipeline in out jenkins file. 

![image](https://user-images.githubusercontent.com/6259479/125304086-32c50580-e325-11eb-8e81-36f44a8b9d63.png)